src/lightning/pytorch/tuner/batch_size_scaling.py

```diff
@@ -32,6 +32,7 @@ def _scale_batch_size(
     init_val: int = 2,
     max_trials: int = 25,
     batch_arg_name: str = "batch_size",
+    is_training: bool = True,
```

Comment:
... and this argument too, but instead capture the `pl_module.training` state at the beginning of this function ...

Reply:
Thanks for the review! `pl_module` is never passed onto the `_scale_batch_size` function but I have accessed the training state from the `trainer`.